rum-injection-installer/installer/nginx_configurator.go

```diff
@@ -0,0 +1,272 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+type NginxConfigurator struct {
+	Version     string
+	ModulesPath string
+}
+
+// Runs 'nginx -v' and looks for the nginx version.
+// Checks that the version is higher than a min version
+func (n *NginxConfigurator) Verify() error {
+	cmd := exec.Command("nginx", "-v")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("nginx is not installed")
+	}
+
+	versionRegex := regexp.MustCompile(`nginx version: nginx/(\d+\.\d+\.\d+)`)
+	matches := versionRegex.FindStringSubmatch(string(output))
+	if len(matches) < 2 {
+		return fmt.Errorf("failed to parse nginx version: %s", output)
+	}
+
+	n.Version = matches[1]
+
+	print("Detected NGINX version: ", n.Version, "\n")
+
+	const minSupportedVersion = "1.22.0"
```

Comment:
The minimum version supported will change over time. The best way to know the minimum supported version is to get the list of module in the latest release.  You can retrieve this list using the Github API.